app/actions/remote/search.ts

```diff
@@ -131,26 +130,11 @@ export const searchPosts = async (serverUrl: string, params: PostSearchParams):
 };
 
 export const searchFiles = async (serverUrl: string, teamId: string, params: FileSearchParams): Promise<FileSearchRequest> => {
-    const operator = DatabaseManager.serverDatabases[serverUrl]?.operator;
-
-    if (!operator) {
-        return {error: `${serverUrl} database not found`};
-    }
-
-    let client: Client;
-    try {
-        client = NetworkManager.getClient(serverUrl);
-    } catch (error) {
-        return {error};
-    }
-
-    let data;
     try {
-        data = await client.searchFiles(teamId, params.terms);
+        const client = NetworkManager.getClient(serverUrl);
+        return await client.searchFiles(teamId, params.terms);
```

Comment:
```suggestion
        return client.searchFiles(teamId, params.terms);
```
Please try the above